contracts/root/RootERC20Predicate.sol

```diff
@@ -71,22 +75,49 @@ contract RootERC20Predicate is Initializable, IRootERC20Predicate {
      * @inheritdoc IRootERC20Predicate
      */
     function deposit(IERC20Metadata rootToken, uint256 amount) external {
+        uint256 expectedBalance = rootToken.balanceOf(address(this)) + amount;
```

Comment:
@drinkcoffee `deposit` and `depositTo` are identical, any reason it isn't DRY'd into a separate private function invoked from these 2?

Reply:
@rzmahmood    tagging you. Agreed @shirren - duplicate code could be in a separate private function.

Reply:
This. is existing code from polygon @shirren . Agreed but I'm not removing it in case they push back due to backwards compatibility issues with integrations.

Reply:
good call, fixed @shirren @drinkcoffee